whats_fresh/whats_fresh_api/templates/story.html

```diff
@@ -123,7 +138,122 @@ <h4 id='error_tag' class='light warning'></h4>
 {% endblock content %}
 
 {% block footer %}
+<script src="//ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.min.js"></script>
+<script src="//ajax.googleapis.com/ajax/libs/jqueryui/1.8.23/jquery-ui.min.js"></script>
+<link href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.8.23/themes/smoothness/jquery-ui.css" rel="stylesheet" type="text/css" media="all" />
 <script>
+
+        if ( window.FormData === undefined )
+            $("#newImage-btn").hide();
```

Comment:
I believe this should do it, yes. If you want to test it yourself, Microsoft offers free web development virtual machines: http://dev.modern.ie/tools/vms/windows/

Otherwise, I can just test it myself in one of those VMs when I get back to Corvallis.